config/webpack/configure-hot-module-replacement.js

```diff
@@ -0,0 +1,45 @@
+var webpack = require('webpack')
+var merge   = require('webpack-merge')
+var path    = require('path')
+
+function configureHotMobuleReplacement(config)
+{
+  let webpackDevServerAddr = process.env['WEBPACK_DEV_SERVER_ADDR'] || 'http://localhost:8080/'
+  config = merge(
+    config,
+    {
+      output: {
+        publicPath: webpackDevServerAddr // necessary for HMR to know where to load the hot update chunks
```

Comment:
I think this should be a PR on webpacker repo as this is generic and will be useful to anybody wanting to use HMR